src/test/crimson/test_socket.cc

```diff
@@ -41,15 +41,15 @@ class SocketFactoryBase
       lo.reuse_address = true;
       factory.listener = seastar::listen(s_addr, lo);
     }).then([this] {
-      return this->container().invoke_on_all([] (auto& factory) {
+      this->container().invoke_on_all([] (auto& factory) {
         // gate accepting
-        seastar::with_gate(factory.shutdown_gate, [&factory] {
+        return seastar::with_gate(factory.shutdown_gate, [&factory] {
```

Comment:
`with_gate(...)` was not chained because L83 would wait for the gate closed. Maybe  you want to suppress the warning, so just add `(void) seastar::with_gate(...)`?

Reply:
ack. fixed and repushed.